src/QsCompiler/Core/Transformations/SyntaxTreeTransformation.fs

```diff
@@ -3,52 +3,37 @@
 
 namespace Microsoft.Quantum.QsCompiler.Transformations.Core
 
+#nowarn "44" // disable deprecation warnings due to the overrides pointing to deprecated methods
```

Comment:
It would be great to add a release `TODO` here to: `TODO: RELEASE 2022-09`.

Reply:
I see multiple of instances of disabling this warning in this PR and since it would be worth adding a `TODO` for all of them since we could forget to re-enable them even after the deprecated methods are deleted.

Reply:
Yes, that is a good idea. I added a TODO: RELEASE 2022-09 tag here, and pointed with a `(*)` to the comment you found/mentioned in your previous comment. That should make things clearer to track.